apps/backend/src/libs/modules/token/token.service.ts

```diff
@@ -0,0 +1,41 @@
+import { type JWTPayload, jwtVerify, SignJWT } from "jose";
+
+import { type Token } from "./token.js";
```

Comment:
this type should be in token/libs/types/types
```suggestion
import { type Token } from "./libs/types/types.js";
```